bin/node/service/src/parachain/service.rs

```diff
@@ -253,6 +254,7 @@ where
                 deny_unsafe,
             )));
             io.extend_with(PubSubApi::to_delegate(PubSubApi::new(pubsub.clone())));
+            io.extend_with(ReqRespApi::to_delegate(robonomics_protocol::reqres::reqresapi::ReqRespApi{}));
```

Comment:
No need to use full path if type already imported

Reply:
fixed